lxd/instance/drivers/driver_lxc.go

```diff
@@ -155,6 +155,7 @@ func lxcCreate(s *state.State, args db.InstanceArgs) (instance.Instance, error)
 			lastUsedDate: args.LastUsedDate,
 			localConfig:  args.Config,
 			localDevices: args.Devices,
+			logger:       logging.AddContext(logger.Log, log.Ctx{"instanceType": args.Type.String, "instanceName": args.Name, "project": args.Project}),
```

Comment:
@kevtheappdev can you replace `instanceName` with `instance` as we've used that elsewhere. Thanks.